src/Transformer/FractalTransformer.php

```diff
@@ -76,7 +76,9 @@ public function transform($response, $transformer, Binding $binding, Request $re
         }
 
         if ($response instanceof EloquentCollection) {
-            $response->load($this->fractal->getRequestedIncludes());
+            $requestedIncludes = $this->fractal->getRequestedIncludes();
+            $eagerLoads = $this->getEagerLoads($transformer, $requestedIncludes);
+            $response->load($eagerLoads);
```

Comment:
Yup! (Or if they're specified in `$defaultIncludes` once this PR gets merged ;) )